spec/models/i18n_content_spec.rb

```diff
@@ -0,0 +1,156 @@
+require 'rails_helper'
+
+RSpec.describe I18nContent, type: :model do
+  let(:i18n_content) { build(:i18n_content) }
+
+  it 'is valid' do
+    expect(i18n_content).to be_valid
+  end
+
+  it 'is not valid if key is not unique' do
+    new_content = create(:i18n_content)
+
+    expect(i18n_content).not_to be_valid
+    expect(i18n_content.errors.size).to eq(1)
+  end
+
+  context 'Scopes' do
+    it 'return one record when #by_key is used' do
+      content      = create(:i18n_content)
+      key          = 'debates.form.debate_title'
+      debate_title = create(:i18n_content, key: key)
+
+      expect(I18nContent.all.size).to eq(2)
+
+      query = I18nContent.by_key(key)
+
+      expect(query.size).to eq(1)
+      expect(query).to eq([debate_title])
+    end
+
+    it 'return all matching records when #begins_with_key is used' do
+      debate_translation = create(:i18n_content)
```

Comment:
A very small thing: I think the test is easier to understand if we don't rely on factory defaults (which by the way could change in the future for whatever reason). Something like `debate_translation = create(:18n_content, key: "debates.title")` makes it easier to understand why `I18nContent.begins_with_key('debates')` should return two records.

Reply:
You're right, @javierm I must have missed that one, thanks! :heart: